lib/Maintenance/Tasks/CleanupFieldcollectionTablesTask.php

```diff
@@ -29,13 +29,23 @@ class CleanupFieldcollectionTablesTask implements TaskInterface
      * @var LoggerInterface
      */
     private $logger;
+    private $mapLowerToCamelCase = [];
 
     /**
      * @param LoggerInterface $logger
      */
     public function __construct(LoggerInterface $logger)
     {
         $this->logger = $logger;
+        
+        $fcPath = PIMCORE_CLASS_DIRECTORY . '/DataObject/Fieldcollection/Data';
```

Comment:
Should this be a constant?

Reply:
Yes, defined in Bootstrap.php:
\Pimcore\Bootstrap::defineConstants()
```
...
        $resolveConstant('PIMCORE_CLASS_DIRECTORY', PIMCORE_PRIVATE_VAR . '/classes');
...
```